controllers/hetznerbaremetalhost_controller.go

```diff
@@ -140,21 +141,18 @@ func (r *HetznerBareMetalHostReconciler) Reconcile(ctx context.Context, req ctrl
 	}
 	if err := r.Client.Get(ctx, hetznerClusterName, hetznerCluster); err != nil {
 		if apierrors.IsNotFound(err) {
+			log.Info("HetznerCluster not found. Requeueing")
 			return reconcile.Result{RequeueAfter: 10 * time.Second}, nil
 		}
 		return reconcile.Result{}, fmt.Errorf("failed to get HetznerCluster: %w", err)
 	}
 
-	log = log.WithValues("HetznerCluster", klog.KObj(hetznerCluster))
```

Comment:
why do you remove this?

Reply:
I removed that, because the logs are hard to read with the human eyes. The signal-to-noise ratio is low.

I think the HetznerCluster does not provide value.

But I can revert that in this PR and create a new one, if you prefer that.

Reply:
In they CAPI docs they say that this is how it should be done. If we have good reasons, then we can remove it. But I would like to have that in a separate PR, including a reasoning why we should NOT follow CAPI here.